cmd/az-log/cmd/node.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// nodeCmd represents the node command
+var nodeCmd = &cobra.Command{
+	Use:   "node",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			fmt.Println("Node is a required argument for the node command")
+		} else {
+			podName := args[0];
+			volumes, nodes, requestIds, afterTime, isWatch, isPrevious := GetFlags(cmd)
+
+			GetLogsByAzDriverPod("azuredisk", podName, volumes, nodes, requestIds, afterTime, isWatch, isPrevious)
```

Comment:
Shouldn't "azuredisk" bet the 2nd parameter ad podName the 1st?

Reply:
We should also make this container name "azuredisk" a constant

Reply:
Thanks. Fixed the argument order and created a constant for "azuredisk".